chart/kubeapps/templates/dashboard/configmap.yaml

```diff
@@ -70,7 +70,7 @@ data:
       "kubeappsCluster": {{ include "kubeapps.kubeappsCluster" . | quote }},
       "kubeappsNamespace": {{ .Release.Namespace | quote }},
       "globalReposNamespace": {{ include "kubeapps.globalReposNamespace" . | quote }},
```

Comment:
This `globalReposNamespace` flag relates only to direct Helm plugin. Shouldn't we also move it to value e.g. `kubeappsapis.pluginConfig.core.packages.v1alpha1.globalPackagingNamespace` ?

Reply:
So that it is in line with the one added for Carvel.

Reply:
This comment should have been added to the `values.yaml` file, but the referenced flag is not available 😄

Reply:
100 % agree, we should move this config to a plugin-aware one.

> kubeappsapis.pluginConfig.core.packages.v1alpha1.globalPackagingNamespace

Did you really mean `core` ? Or `helm` and `flux` separately?  I don't think it should be a core config, as in each plugin it can be different.

Reply:
We don't yet have the specific `helm` config path, that is why mentioned the core. But totally +1 to creating the specific `directhelm`/`helm` config like we have for the other plugins, aggregating the ones from `core`.
So it would be `kubeappsapis.pluginConfig.helm.packages.v1alpha1.globalPackagingNamespace`.

Reply:
I've been looking into it... but modifying it would imply backward incompatible changes. Currently, we are defaulting to "kubeapps namespace + suffix" in `apprepos ctrl`, `asset-syncer` and `kubeappsapis`, however, it's being done by defining  a template constant in helm.

So my approach is: if the user selects a `helm....globalPackagingNamespace`, then use it. Otherwise, fall back to the old behavior. This way, we can avoid releasing a new major chart version with an explicit upgrade path.